types/bluebird/index.d.ts

```diff
@@ -573,6 +574,13 @@ declare class Bluebird<R> implements PromiseLike<R>, Bluebird.Inspection<R> {
    */
   all(): Bluebird<never>;
 
+  static allSettled<T1, T2, T3, T4, T5>(values: [Resolvable<T1>, Resolvable<T2>, Resolvable<T3>, Resolvable<T4>, Resolvable<T5>]): Bluebird<ResultArrayToInspection<[T1, T2, T3, T4, T5]>>;
```

Comment:
Could you please move these definitions down to where all the other static definitions are?
I'd suggest moving them to 865, after the definitions for the static `all()` method.

Reply:
@lhecker done